docs/src/Dropdown.doc.js

```diff
@@ -231,54 +212,425 @@ card(
         type: 'React.ChildrenArray<React.Element<typeof DropdownItem>>',
         required: true,
         description: 'Any Dropdown.Items to be rendered',
-        href: 'withSections',
       },
       {
         name: 'label',
         type: 'string',
         required: true,
-        description: 'Label for the section',
-        href: 'withSections',
+        description: 'Label for the section. See the [Sections](#Sections) variant for more info.',
       },
     ]}
   />,
 );
 
 card(
-  <Card
-    description={`
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Dropdown when additional functionality is needed such as subtext, custom header etc."
```

Comment:
additional functionality compared to... SelectList? I'd specify compared to what. What are the basic components that can replace Dropdown if additional functionality is not needed.